spec/controllers/bills_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+describe BillsController do
+
+  describe "GET #index" do
+    let!(:bill) { FactoryGirl.create(:bill) }
```

Comment:
Sorry, my bad -- you can totally use a `let` outside the `describe "GET"` just on a line by itself, not in a before block. If you do that, the `let!` statement in the destroy test still needs to be a `let!` since that test doesn't work if `:bill` is lazy evaluated.